chart/bagetter/Chart.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v2
+name: bagetter
+description: A Helm chart for BaGetter nuget package server
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "1.0.4"
```

Comment:
What about increasing this version number? We use the Git tags and GitHub actions to set the version of BaGetter. I think you have to edit the `.github/workflows/release.yml` and add a new job where the source code of the file (version number) gets replaced by the action.

Reply:
Yes. I was thinking about that, too. 
I need to look into it...